tests/parametric_study/test_parametric_study.py

```diff
@@ -1143,6 +1316,110 @@ def test_add_inputs_assigns_unspecified_microstructure_params_correctly(
     assert np.isnan(df.loc[0, ps.ColumnNames.RANDOM_SEED])
 
 
+@pytest.mark.parametrize(
+    "input_status, expected_len",
+    [
+        (SimulationStatus.PENDING, 3),
+        (SimulationStatus.SKIP, 3),
+        (SimulationStatus.ERROR, 0),
+        (SimulationStatus.COMPLETED, 0),
+    ],
+)
+def test_add_inputs_only_adds_entries_with_simulation_status_pending_or_skip(
```

Comment:
`add_inputs()` should raise a `ValueError` exception if `status` is not PENDING or SKIP. Example:
```
    # act, assert
    with pytest.raises(ValueError, match="some portion of exception message"):
        study.add_inputs(inputs, status=input_status)
```
This would be a separate test, similar to this one but with only ERROR and COMPLETED status' as the parameters. You'll need to remove ERROR and COMPLETED parameters from this test.